documentation/milestones/individual_milestones/communication_platforms.md

```diff
@@ -141,65 +143,72 @@ Ideally, the tool would have the ability to designate some users as direct edito
+  - [ ] Users can access a live transcript during the video call
+- [ ] Wiki is live with the following conditions satisfied:
+  - [ ] At least 1 document has been added to the wiki (preferably the onboarding instructions for these tools)
+  - [ ] Without logging in, members of the public can see wiki documents.
+  - [ ] Users can see history of changes to all documents.
+  - [ ] Web analytics have been integrated into the wiki and administrators can report on unique users, page views, etc.
 - [ ] Email listserv is live with the following conditions satisfied
   - [ ] At least one email has been sent out to the listserv
   - [ ] Any member of the listserv can send emails to the entire listserv
   - [ ] Any member of the listserv can reply all to an email
   - [ ] Any member of the listserv can reply directly to an email sender
 - [ ] Shared email inbox is live with the following conditions satisfied
   - [ ] At least 1 email has been received
-- [ ] Video conference is live with the following conditions satisfied
-  - [ ] At least one public meeting has been hosted
-  - [ ] At least one public meeting has been recorded, with recording and transcript uploaded to the wiki
-  - [ ] Users can join the call via video or phone
-  - [ ] Users can access a live transcript during the video call
 
 ### Proposed metrics for measuring goals/value/definition of done
 
 TODO: See if we can reduce the list of metrics to track
 
-1. Chat Metrics
+1. Ticket Tracking Metrics
+   1. Total story points assigned per sprint
+   2. Total story points completed per sprint
+2. Chat Metrics
    1. Number of monthly active users in chat, total
    2. Number of monthly active users in chat, external
    3. Weekly volume of Chat messages
-2. Wiki Metrics
+3. Calendar Metrics
+   1. Number of public meetings per quarter
+   2. Number of external users attending public meetings
+4. Video Conference
+   1. Number of attendees, total (public meetings)
+   2. Number of attendees, external (public meetings)
+5. Wiki Metrics
    1. Number of monthly active users in wiki, total
    2. Number of monthly active users in wiki, visitors
    3. Weekly volume of wiki edits/contributions
-3. Calendar Metric: Number of external users attending public meetings
-4. Listserv Metrics
+6. Listserv Metrics
    1. Number of subscribed users in listserv, total
    2. Number of subscribed users in listserv, external
    3. Average number of responses in listserv thread
-5. Shared Inbox Messages
+7. Shared Inbox Metrics
    1. Weekly volume of messages to shared inbox
    2. Average response time for a message
-6. Video Conference
-   1. Number of attendees, total (public meetings)
-   2. Number of attendees, external (public meetings)
 
 ### Destination for live updating metrics
 
-Not yet known.
+Page on the public wiki. **Note:** This will likely change once we deliver [the Public Measurement Dashboard milestone](../milestone_short_descriptions.md#public-measurement-dashboards)
 
 ## Planning
 
```

Comment:
not to be hoist by my own petard here, but I think we're missing some required sections (that yours truly left out in the first draft of this). 

These may actually be useful here to think about threat vectors, services (like Slack/chat?) going to PROD, etc! 

We may also want to think a bit about integrating services, such as hooking our calendar app and tickets app up to our chat app, etc etc.

Reply:
@lucasmbrown-usds Good point! I'll add these into the milestone doc along with some of the new sections proposed in the meeting today (i.e. doc sharing and retro tool).